### ISSUE TITLE
🚀 Feature Request: Simplified Method for Changing Label Names in YOLOv5 Model

### ISSUE THREAD

**Issue by osalhi-kali** [ultralytics/yolov5#13036]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar feature requests.

### Description

Background

Many users have reported issues with misspelled label names in their trained YOLOv5 models. Often, they are unaware of how to update these labels easily and end up retraining the entire model, which is impractical and time-consuming. Issues such as [#12156](https://github.com/ultralytics/yolov5/issues/12156) and [#3577](https://github.com/ultralytics/yolov5/issues/3577) highlight the need for a straightforward solution to update label names in an existing YOLOv5 model.
Proposed Solution

I propose adding a guide and utility script to the YOLOv5 documentation that explains how to change label names without retraining the model. This solution leverages PyTorch to load, update, and save the model with new label names.
Implementation Guide

The following Python script demonstrates how to change the label names of a YOLOv5 model:
```python
import torch

# Load your trained YOLOv5 model
model = torch.load("path/to/best.pt")

# Define New Label Names
model['model'].names = ["name1", "name2", ...]

# Save the updated model
torch.save(model, "path/to/update_best.pt")
```
Adding this guide to the YOLOv5 documentation will greatly benefit users who face issues with label name errors. It addresses a common problem and provides a practical, efficient solution.

Thank you for considering this feature request.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by osalhi-kali** [comment]
import torch

# Load your trained YOLOv5 model
model = torch.load("path/to/best.pt")

# Define New Label Names
model['model'].names = ["name1", "name2"]

# Save the updated model
torch.save(model, "path/to/update_best.pt")

**Comment by glenn-jocher** [comment]
Hello! Thanks for your feature request and for providing a detailed proposal on how to simplify the process of changing label names in YOLOv5 models. 🚀

Your suggestion to add a utility script and guide for updating label names directly in the model file without retraining is indeed valuable. This could save users a lot of time and effort, especially when dealing with minor errors like misspellings.

We appreciate your initiative in drafting a potential solution. I'll bring this to the team's attention so we can discuss the feasibility of integrating this feature into our documentation or tools. Your contribution to making YOLOv5 more user-friendly is invaluable, and we encourage you to stay tuned for updates or consider submitting a PR as you mentioned!

Thanks again for your input and support for the YOLOv5 community! 🌟
